lib/manager/kustomize/extract.ts

```diff
@@ -107,7 +107,7 @@ export function extractImage(image: Image): PackageDependency | null {
 }
 
 export function parseKustomize(content: string): Kustomize | null {
-  let pkg = null;
+  let pkg: Kustomize | null | undefined = null;
```

Comment:
```suggestion
  let pkg: Kustomize | null = null;
```

I think `load` will never return `undefined`